### PR TITLE
13.0 mig account invoice tax required

### PR DESCRIPTION
**Pull request by jvpascual**
Migration to 13.0

### CODE REVIEW

File: account_invoice_tax_required/tests/test_account_invoice_tax_required.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Copyright 2016 - Tecnativa - Angel Moya <odoo@tecnativa.com>
```
Comment (kongrattapong) [inline account_invoice_tax_required/tests/test_account_invoice_tax_required.py:1]: change file name to **test_account_move_tax_required.py**

File: account_invoice_tax_required/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import test_account_invoice_tax_required
```
Comment (kongrattapong) [inline account_invoice_tax_required/tests/__init__.py:3]:
```suggestion
from . import test_account_move_tax_required
```

File: account_invoice_tax_required/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+To use this module, you need to:
+
+#. Go to Accounting (or Invoicing) > Sales > Customer Invoices.
+#. Create a new invoice.
+#. Insert a new invoice line without taxes.
+#. Press on "Validate".
```
Comment (kongrattapong) [inline account_invoice_tax_required/readme/USAGE.rst:6]:
```suggestion
#. Press on "Post".
```

File: account_invoice_tax_required/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,7 @@
+To use this module, you need to:
+
+#. Go to Accounting (or Invoicing) > Sales > Customer Invoices.
```
Comment (kongrattapong) [inline account_invoice_tax_required/readme/USAGE.rst:3]:
```suggestion
#. Go to Invoicing > Customers > Invoices.
```

File: account_invoice_tax_required/models/account_invoice.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            self.with_user(SUPERUSER_ID).env["ir.module.module"].search([
+                ("state", "in", ["to install", "to upgrade"]),
+                ("demo", "=", True),
+            ]),
+            # Avoid breaking unaware addons' tests by default
+            config["test_enable"],
+        ))
+        if force_test or not skip_test:
+            self._test_invoice_line_tax()
+        return super(AccountMove, self).action_invoice_open()
```
Comment (kongrattapong) [inline account_invoice_tax_required/models/account_invoice.py:43]:
```suggestion
        return super(AccountMove, self).action_post()
```

File: account_invoice_tax_required/models/account_invoice.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+            if not invoice_line.tax_ids:
+                error_string = error_template % (invoice_line.name)
+                errors.append(error_string)
+        if errors:
+            raise UserError(
+                _('%s\n%s') % (_('No Taxes Defined!'),
+                               '\n'.join(x for x in errors))
+            )
+
+    def action_invoice_open(self):
```
Comment (kongrattapong) [inline account_invoice_tax_required/models/account_invoice.py:29]:
```suggestion
    def action_post(self):
```

File: account_invoice_tax_required/models/__init__.py [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import account_invoice
```
Comment (kongrattapong) [inline account_invoice_tax_required/models/__init__.py:3]:
```suggestion
from . import account_move
```
then change account_invoice.py to account_move.py

**Review by kongrattapong — CHANGES_REQUESTED** [review]
@jvpascual

**Comment by AaronHForgeFlow** [comment]
Hi @jvpascual Do you plan to continue working on this PR?

**Comment by kittiu** [comment]
May I close this as it seem inactive with another PR already approved.

**Pull request closed without merging**